history-service/init.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE submissions (
+	submission_id SERIAL PRIMARY KEY,
+	question_id INT NOT NULL,
+	runtime INT NOT NULL,
```

Comment:
Mentioned before, but we are not recording runtime as we do not provide test cases for users to run their code against.